contents/tutorials/nextjs-monitoring.md

```diff
@@ -0,0 +1,217 @@
+---
+title: How to set up Next.js monitoring
+date: 2023-05-23
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/nextjs-analytics.png
+tags: ['configuration', 'session replay', 'insights']
+---
+
+Monitoring a Next.js app ensures the best possible experience for your users. It ensures the optimizations Next.js provides are working as well as possible, and minmizes errors hurting user experience.
```

Comment:
Potentially stupid question: What exactly do we mean by "monitoring"? Error monitoring? Usage monitoring Something else? It's not clear to me what monitoring is

Reply:
A bit of both, performance monitoring (loading speeds, web vitals), usage monitoring (being used and lack of issues, errors). Rewrote.